aztec_macros/src/lib.rs

```diff
@@ -31,10 +31,15 @@ impl MacroProcessor for AztecMacro {
     }
 }
 
+const FUNCTION_TREE_HEIGHT: u32 = 5;
+const MAX_CONTRACT_FUNCTIONS: usize = 2_usize.pow(FUNCTION_TREE_HEIGHT);
```

Comment:
Should we have `circuits.js/src/scripts/constants.in.ts` spit out a .rs file in noir with this data?

Reply:
yes - very good point

Reply:
https://github.com/AztecProtocol/aztec-packages/issues/4129